### PR TITLE
use inverted colour for anchors on dark backgrounds

### PR DESCRIPTION
**Pull request by webb04**
## Changes

- Improve contrast of anchors on media pages 
- Use strong elements to keep inline display

## Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |
|  |  |

### CODE REVIEW

File: src/components/anchor.tsx [hunk 1]
```diff
@@ -18,18 +18,32 @@ interface Props {
...
+            return css`
+                color: ${kicker};
+                border-bottom: 0.0625rem solid ${neutral[86]};
+            `
+    }
+}
+
 const Anchor: FC<Props> = ({ format, children, href, className }: Props) =>
-    <a css={[styles(getPillarStyles(format.pillar).kicker), className]} href={href}>
+    <a css={[styles, className, colour(format)]} href={href}>
```
Comment (JamieB-gu) [inline src/components/anchor.tsx:46]:
```suggestion
    <a css={[styles, colour(format), className]} href={href}>
```

File: src/components/figCaption.tsx [hunk 2]
```diff
@@ -76,11 +77,11 @@ const captionElement = (format: Format) => (node: Node, key: number): ReactNode
     const children = Array.from(node.childNodes).map(captionElement(format));
     switch (node.nodeName) {
         case 'STRONG':
-            return <h2 css={captionHeadingStyles} key={key}>{children}</h2>;
+            return <strong css={captionHeadingStyles} key={key}>{children}</strong>;
```
Comment (JamieB-gu) [inline src/components/figCaption.tsx:80]: The choice of `h2` here was for semantic reasons (plus they're `h2` elements on dotcom). If the aim is to get inline styles can we use `display: inline` instead?

**Comment by frankie297** [comment]
You might want to check with @benwuersching but I'm pretty sure we aren't supposed to be using pillar colours for anchors anymore? I think it's supposed to be the same colour as the article copy with an underline (same colour)

**Comment by frankie297** [comment]
Like this....
image

**Comment by benwuersching** [comment]
@webb04 @JamieB-gu @frankie297 For a general look at the design of dark mode media pages (and links on them), please see here:

Galleries:
https://app.zeplin.io/project/58ee27a3e08030c8234f88a3/dashboard?seid=5d67f6e1f555509eedb2b361

Podcasts:
https://app.zeplin.io/project/58ee27a3e08030c8234f88a3/dashboard?seid=5d694f5ff555509eedc232e4

Videos:
https://app.zeplin.io/project/58ee27a3e08030c8234f88a3/dashboard?seid=5d67f67646e50e9b4232f97d

**Comment by webb04** [comment]
Good spot, I know it's like that on dark mode. Should be easy for us to change either way

| Current app | Current app dark mode |
| --- | --- |
|  |  |

**Comment by frankie297** [comment]
> For a general look at the design of dark mode media pages (and links on them), please see here:

@benwuersching Does this only apply to Dark Mode then?

**Comment by benwuersching** [comment]
@frankie297 @webb04 Ah yes! In light mode, we do want to keep the pillar colour for links/anchors etc. if currently used. If we currently use any colours that are hard to read on black background, then let's find a more suitable colour. It should come, however, from the range of options within the pillar in question.

**Comment by webb04** [comment]
Is this ok to go @frankie297 @JamieB-gu.

I don't think we have any dark mode styles yet so I'll save them for another ticket.

**Pull request merged**